### ISSUE TITLE
Deminify js code in <script> tags.

### ISSUE THREAD

**Issue by nandod** [benmmurphy/jsdeminifier_xpi#2]
Hello,
it would be nice if jsdeminifier could deminify js code in < script > tags inside text/html responses. I tried to add text/html to the list of supported mime types but it tries to deminify everything thus breaking the HTML.

**Comment by benmmurphy** [comment]
This is a nice idea but it is quite tricky do do :(

**Comment by nandod** [comment]
I guess so. :-( It would need to parse the stream and only pass what's between <script> and </script> to the beautifier. Or an alternate beautifier that doesn't mangle HTML code, which I am not aware of.

**Comment by nandod** [comment]
Between < script > and </ script > I meant.

**Comment by benmmurphy** [comment]
this is true. sounds a bit easier than I thought it would be. i'll start work on it after i have closed off https://github.com/benmmurphy/jsdeminifier_xpi/issues/3

**Comment by caesarsol** [comment]
hello, any news on this feature? :)
if I paste HTML code on jsbeautify.com it can manage the HTML, maybe you could update the algorithm?
then it would be enough to insert `text/html` in the `extensions.jsdeminifier.contenttypes` config, to pass all the HTML to the deminifier!
thanks!
